docs/automerge_synopsis.md

```diff
@@ -0,0 +1,128 @@
+## Automerge Synopsis
```

Comment:
Can you move this to `docs/README.md` so that people can see it when they go to `docs`?